### ISSUE TITLE
broken release 0.12.0

### ISSUE THREAD

**Issue by camaeel** [jtblin/kube2iam#381]
It seems that release https://github.com/jtblin/kube2iam/releases/tag/0.12.0 is broken: it has wrong title, and there are no image containers in dockerhub.com

**Comment by jtblin** [comment]
Fixed the title for https://github.com/jtblin/kube2iam/releases/tag/0.12.0

**Comment by tropnikovvl** [comment]
Hi @jtblin ,
Can you also release a helm chart for it?

**Comment by camaeel** [comment]
@jtblin, it seems the image for 0.12.0 is not pullable:

```
❯ docker pull jtblin/kube2iam:0.12.0 
Error response from daemon: manifest for jtblin/kube2iam:0.12.0 not found: manifest unknown: manifest unknown
```

**Comment by jangrewe** [comment]
And the `latest` tag still points to `0.11.1`.

**Comment by g-sree** [comment]
@jtblin Image tag 0.13.0 not pushed to docker-hub

```
$ docker pull jtblin/kube2iam:0.13.0
Error response from daemon: manifest for jtblin/kube2iam:0.13.0 not found: manifest unknown: manifest unknown
```

```
Failed to pull image "jtblin/kube2iam:0.13.0": rpc error: code = NotFound desc = failed to pull and unpack image "jtblin/kube2iam:0.13.0": failed to resolve reference "jtblin/kube2iam:0.13.0": jtblin/kube2iam:0.13.0: not found
```

looks like tag build has no workflow

https://app.circleci.com/pipelines/circleci/Tv5Xa331HXLx98iuW72Ug4/Po3PqWxjiikmpntHfiZoXW

<img width="1193" alt="Image" src="https://github.com/user-attachments/assets/d8f5d44c-7351-4292-b7b2-1456742807b6" />

**Comment by jtblin** [comment]
Yes I know. Not sure why the ci pipeline is not triggering on tag and not pushing the docker images with the appropriate tag. Will look into this but let me know if you know why.

**Comment by g-sree** [comment]
> Yes I know. Not sure why the ci pipeline is not triggering on tag and not pushing the docker images with the appropriate tag. Will look into this but let me know if you know why.

as per this circleci config https://github.com/jtblin/kube2iam/blob/master/.circleci/config.yml#L83-L86, the docker push happens based on branch filter `master` or `release-<tag>`

So if you create a branch with `release-0.13.0` from latest master branch it should work.. or you should change cirlcle ci config to change to tag filter 

```
          filters:
            tags:
              only: /^v.*/
```

something like this

**Comment by jtblin** [comment]
Thanks, the problem is actually that `CIRCLE_TAG` env variable is not set so the release is never triggered, see https://support.circleci.com/hc/en-us/articles/360020342494-The-built-in-environment-variable-CIRCLE-TAG-is-not-being-set

I changed the config to build on all branches and tags so now pushing a new tag should push to docker.

**Comment by jtblin** [comment]
I don't think I'll be able to push `0.12.0`, workaround is to use the hash: `docker pull jtblin/kube2iam:c23584c` instead.

[0.13.0](https://hub.docker.com/repository/docker/jtblin/kube2iam/tags/0.13.0/sha256:ffb36bbed931a6478ea4e9cb720766ce28c392458c2c25793b3d1a1db0bab76a) and [latest](https://hub.docker.com/repository/docker/jtblin/kube2iam/tags/latest/sha256:ffb36bbed931a6478ea4e9cb720766ce28c392458c2c25793b3d1a1db0bab76a) are now pushed to Dockerhub so closing this issue.
